### ISSUE TITLE
Linux Launcher fails to run (chrome sandbox)

### ISSUE THREAD

**Issue by conor-oneill-2** [Revolutionary-Games/Thrive-Launcher#151]
I downloaded the Linux Launcher file (I'm running Arch Linux), but couldn't get it to run. I've marked the file as an executable, but I still get the following error message: 

```
Cannot open /proc/self/exe: Permission denied
Failed to get fs offset for /proc/self/exe
```

Since it looked like a permissions error, I tried running the file as root, and got a different error message:

```
[0321/175816.376189:FATAL:electron_main_delegate.cc(298)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.
Trace/breakpoint trap
```

Looking through the link provided, I didn't see anything that looked useful.

Googling around, the only suggested solution I found for that error message was to reset permissions for the fusermount file ( from here: https://github.com/keepassxreboot/keepassxc/issues/5138 ), but I tried that and that didn't work either.

**Comment by hhyyrylainen** [comment]
I think you ran into the chrome sandbox issue. Which got reported to the wrong repo (it was talked about in the main Thrive repo): https://github.com/Revolutionary-Games/Thrive/issues/749

I think the solution is to give the right permissions for the sandbox executable, update your kernel, or run the launcher with the `--no-sandbox` flag.

**Comment by conor-oneill-2** [comment]
Trying to run with the ``--no-sandbox`` flag just gave another error message. I'm confident the kernel is up to date, and I looked to see if anything funny was happening with the permissions or namespaces, but couldn't see anything unusual.

However, I found another way around that anyone else struggling with the installer might be interested in - I skipped installing the launcher altogether and just installed the Thrive 7zip file directly from here ([https://github.com/Revolutionary-Games/Thrive/releases/tag/v0.5.7](https://github.com/Revolutionary-Games/Thrive/releases/tag/v0.5.7)). After unzipping, everything worked perfectly out of the box.

I don't know whether to consider the issue closed for that reason, since technically I didn't figure out the source of the issue - I'll leave that up to hhyyrylainen or one of the other devs to decide. But I'm happy with the workaround, since I got the game running.
